### PR TITLE
simplify beatmap pack installation instructions

### PR DESCRIPTION
**Pull request by Samaoo**
- simplify installation instructions 
- rephrase a few things

### CODE REVIEW

File: resources/lang/en/beatmappacks.php [hunk 1]
```diff
@@ -12,14 +12,10 @@
         'blurb' => [
             'important' => 'READ THIS BEFORE DOWNLOADING',
             'instruction' => [
-                '_' => "Installation: Once a pack has been downloaded, extract the .rar into your osu! Songs directory.
-                    All songs are still .zip'd and/or .osz'd inside the pack, so osu! will need to extract the beatmaps itself the next time you go into Play mode.
-                    :scary extract the zip's/osz's yourself,
-                    or the beatmaps will display incorrectly in osu! and will not function properly.",
-                'scary' => 'Do NOT',
```
Comment (peppy) [inline resources/lang/en/beatmappacks.php:19]: I don't think this part should change. The process is still the same.

File: resources/lang/en/beatmappacks.php [hunk 2]
```diff
@@ -12,14 +12,10 @@
...
+                '_' => 'Installation: Once a pack has been downloaded, extract the contents of the pack into your osu! Songs directory.',
             ],
             'note' => [
-                '_' => 'Also note that it is highly recommended to :scary, since the oldest maps are of much lower quality than most recent maps.',
```
Comment (peppy) [inline resources/lang/en/beatmappacks.php:22]: Not against this line change though.

**Review by peppy — CHANGES_REQUESTED** [review]

File: resources/lang/en/beatmappacks.php (hunk 1 above)
Comment (Samaoo) [reply resources/lang/en/beatmappacks.php:19]: I think all the user should know is that they should extract the beatmappack to the osu songs folder. If you don't want users to extract osz files manually, I think the best way to do that is just to not tell them that they can be extracted manually, the way it's currently phrased just overcomplicates things in my opinion.

Comment (peppy) [reply resources/lang/en/beatmappacks.php:19]: Will wait for more opinions on this

Comment (TicClick) [reply resources/lang/en/beatmappacks.php:19]: agree with Samaoo (not to prevent people from extracting `.osz`s, but because that's irrelevant here)

**Review by peppy — APPROVED** [review]
Let's go with this.

**Pull request merged**